go/mysql/auth_server_static.go

```diff
@@ -41,13 +41,15 @@ var (
 	mysqlAuthServerStaticFile           string
 	mysqlAuthServerStaticString         string
 	mysqlAuthServerStaticReloadInterval time.Duration
+	mysqlServerFlushDelay               time.Duration
```

Comment:
this needs the default value here:

```go
mysqlServerFlushDelay = 100 * time.Millisecond


....

OnParseFor("...", func(...) {
    fs.DurationVar(&mysqlServerFlushDelay, "mysql_server_flush_delay", mysqlServerFlushDelay, "<usage text goes here>")
}
```

Reply:
I've changed this. But it's not really required is it? It's not clear to me why we would handle this particular flag value differently — knowing this will help me with the others.

Reply:
it's required for any package that imports (uses the default value) without exposing the flag, which does happen from time to time